docs/sdks/uid2-sdk-ref-python.md

```diff
@@ -66,58 +63,111 @@ The package is published in this location:
 - [https://pypi.org/project/uid2-client/](https://pypi.org/project/uid2-client/)
 
 ## Initialization
+The initialization step depends on the role, as shown in the following table.
+
+| Role	     | Create Instance of Class	 | Link to Instructions                          |
+|:----------|:--------------------------|:----------------------------------------------|
+| DSP       | `BidstreamClient`         | [Usage for DSPs](#usage-for-dsps)             |
+| Sharer    | `SharingClient`           | [Usage for Sharers](#usage-for-uid2-sharers)  |
+| Publisher | `Uid2PublisherClient`     | [Usage for Publishers](#usage-for-publishers) |
 
-The initialization function configures the parameters necessary for the SDK to authenticate with the UID2 service. It also allows you to configure retry intervals in the event of errors.
 
-| Parameter | Description | Recommended Value |
-| :--- | :--- | :--- |
-| `endpoint` | The endpoint for the UID2 service. | N/A |
-| `authKey` | The authentication token that belongs to the client. For access to UID2, see [Contact Info](../getting-started/gs-account-setup.md#contact-info). | N/A |
+You will need to provide the values necessary for the SDK to authenticate with the UID2 service.
+
+| Parameter    | Description                                                                               |
+|:-------------|:------------------------------------------------------------------------------------------|
+| `endpoint`   | The endpoint for the UID2 service. See [Environments](../getting-started/gs-environments) |
```

Comment:
Line #79 -- just add a period at the end:

| `endpoint` | The endpoint for the UID2 service. See [Environments](../getting-started/gs-environments). |